python/src/trezorlib/transport/protocol.py

```diff
@@ -31,6 +33,10 @@
 
 LOG = logging.getLogger(__name__)
 
+DEFAULT_READ_TIMEOUT = os.environ.get("TREZOR_DEFAULT_READ_TIMEOUT")
```

Comment:
what is the usecase for this?
(also i'm surprised the typechecker is not shouting at you about the type of `DEFAULT_READ_TIMEOUT`

Reply:
I plan to set it to ~10 seconds, in order to recover from lost USB packet scenario (in case the whole response fits in a single packet) without waiting for the test case timeout ([which is currently set to 10 minutes](https://github.com/trezor/trezor-firmware/blob/e4ed101d01bf0815392c36d3c99292380ac8cad1/.github/workflows/core-hw.yml#L37)).

Reply:
Regarding typechecker - I think it's OK since `DEFAULT_READ_TIMEOUT` is initially `str | None`, and after the `if` statement it becomes `float | None`...

If preferable, I can change it to:
```
DEFAULT_READ_TIMEOUT = float(os.environ.get("TREZOR_DEFAULT_READ_TIMEOUT", "inf"))
```
so there is no need to use `None` to indicate infinite timeout. 

WDYT?

Reply:
from slack convo: let's make this instead an internal private variable and have conftest explicitly set its value

Reply:
Fixed in https://github.com/trezor/trezor-firmware/pull/4748/commits/75cc399e80d600494cc3510cf61fc3123f86a946 (also added the timeout value to the exception message).

Reply:
Pushed 0e2339090b83703d4ed76d45e3911b4a677a2190 (following https://github.com/trezor/trezor-firmware/commit/75cc399e80d600494cc3510cf61fc3123f86a946#commitcomment-153830441).

Reply:
Increased the default timeout to 50s, due to `tests/click_tests/test_autolock.py::test_dryrun_enter_word_slowly`.